classes/item.rb

```diff
@@ -1,29 +1,34 @@
-require 'date' # Require the Date class
-
 class Item
-  attr_accessor :id, :genre, :author, :label, :publish_date
-  attr_reader :archived
+  attr_accessor :title
+  attr_reader :id, :publish_date, :archived
 
-  def initialize(genre, author, label, publish_date, archived)
-    @id = Random.rand(1..1000)
-    @genre = genre
-    @author = author
-    @label = label
+  def initialize(id, title, publish_date)
+    @id = id
+    @title = title
     @publish_date = publish_date
-    @archived = archived
+    @archived = false
   end
 
   def can_be_archived?
-    return false if @publish_date.nil?
+    !archived
+  end
```

Comment:
can_be_archived?() in the Item class
should return true if published_date is older than 10 years
otherwise, it should return false